cfpb/cfgov-refresh
# Add flagged Qualtrics website feedback snippet for Ask CFPB survey

## Additions

- JavaScript snippet for the FY2020 Q4 Ask CFPB feedback survey added behind a new `ASK_SURVEY_INTERCEPT` feature flag
- Adds `*.qualtrics.com` to CSP allow lists.


## How to test this PR

1. It's running on the cfgov-adhoc-intercept demo server
1. Visit any of the Money Topic portal pages other than Debt Collection and Savings, or most any Ask CFPB answer page
1. Wait 5 seconds and see the Feedback tab appear
1. Scroll the page and see the survey slide out
1. Take the survey


## Screenshots

![Survey window slid out on the Auto Loans portal page](https://user-images.githubusercontent.com/1044670/85428836-c88bc800-b54b-11ea-822e-1d770fe8e778.png)


## Checklist
